websites/docs-smoke-test/src/content/index.mdx

```diff
@@ -29,3 +30,13 @@ title: Explore how to build a documentation website for commercetools
   <Card>Avengers: Age of Ultron</Card>
   <Card>Avengers: Infinity War</Card>
 </Cards>
+
+# Logged in or logged out content
+
+<IsUserLoggedIn>
+ ## Welcome back!
+</IsUserLoggedIn>
+
+<IsUserLoggedOut>
+ ## Please log in
+</IsUserLoggedOut>
```

Comment:
Would it be more readable or "template language like"  if it were `<IfUserLoggedIn>`  vs. `<IsUserLoggedIn>` ?

Reply:
Yes, agreed